biomage/experiment/utils.py

```diff
@@ -235,4 +170,6 @@ def add_env_user_to_experiment(cfg):
 
     user_name = get_cognito_username(email=email)
 
-    return add_user_to_rbac(user_name=user_name, cfg=cfg)
+    add_user_to_rbac(user_name=user_name, cfg=cfg)
```

Comment:
I would rather not use named arguments here

Reply:
why not, I'd rather use them almost always

Reply:
Because the parameters are named exactly the same as the arguments I just think it looks weird